action.php

```diff
@@ -68,6 +68,28 @@ function _handle_function_submit(&$event, $param) {
             msg($this->getlang('email_err'), -1);
             $err=true;
         }
+
+        $response = $_POST["g-recaptcha-response"];
+        $url = 'https://www.google.com/recaptcha/api/siteverify';
+        $data = array(
+            'secret' => 'YOUR_SECRET',
```

Comment:
Where does this secret come from? 
Also, is this recaptcha v2 or v3?

Reply:
Secret comes from your own authenticated captcha, the task said to not reveal mine on the PR. It is captcha v2 checkbox